code/game/objects/items/weapons/shards.dm

```diff
@@ -25,8 +24,7 @@
 									"<span class='danger'>[user] is slitting [user.p_their()] throat with [src]! It looks like [user.p_theyre()] trying to commit suicide.</span>"))
 		return BRUTELOSS
 
-/obj/item/shard/Initialize(mapload)
-	. = ..()
+/obj/item/shard/proc/initial_icon_state()
```

Comment:
A proc usually has a verb in the name. Since you're doing, setting or getting something.